app/javascript/pages/country/components/widget-tree-cover-loss-areas/widget-tree-cover-loss-areas-component.js

```diff
@@ -0,0 +1,84 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { PieChart, Pie, Legend, Tooltip, Cell }from 'recharts';
+import numeral from 'numeral';
+
+import TooltipChart from '../tooltip-chart/tooltip-chart';
+import WidgetHeader from '../widget-header/widget-header';
+
+class WidgetTreeLossAreas extends PureComponent {
+  componentDidMount() {
+    const { setInitialData } = this.props;
+    setInitialData(this.props);
+  }
+
+  render() {
+    const {
+      isLoading,
+      countryData,
+    } = this.props;
+
+    const pieCharDataDistricts = [
+      { name: 'Minas Gerais', value: 1200, color: '#510626' },
+      { name: 'Bahia', value: 1100, color: '#730735' },
+      { name: 'Amazonas', value: 900, color: '#af0f54' },
+      { name: 'Maranhao', value: 550, color: '#f5247e' },
+      { name: 'Distrito Federal', value: 464, color: '#f3599b' },
+      { name: 'Ceará', value: 460, color: '#fb9bc4' },
+      { name: 'Espírito Santo', value: 440, color: '#f1c5d8' },
+      { name: 'Goiás', value: 420, color: '#e9e7a6' },
+      { name: 'Maranhão', value: 300, color: '#dad781' },
+      { name: 'Mato Grosso', value: 203, color: '#cecb65' },
+      { name: 'Other Districts', value: 3000, color: '#e9e9ea' }
+    ];
+
+    if (isLoading) {
+      return <div>loading!</div>
+    } else {
+      return (
+        <div className="c-widget c-widget-tree-cover-loss-areas">
+          <WidgetHeader title={`AREAS WITH MOST TREE COVER LOSS IN ${countryData.name}`} />
+          <div className="c-widget-tree-cover-loss-areas__container">
+            <div className="c-widget-tree-cover-loss-areas__chart">
+              <h3 className="title">Total Tree cover loss</h3>
+              <p className="date">(2011 - 2016)</p>
```

Comment:
On the widget options you can change the min and max year, so this should be a variable stored on redux